tests/test_configuration.py

```diff
@@ -13,7 +13,7 @@ def test_refresh_rate(self):
 
     def test_github_repositories(self):
         repositories = self.configuration.github_repositories()
-        self.assertEqual(3, len(repositories))
+        self.assertEqual(5, len(repositories))
```

Comment:
Does this check make sense?

This pull request shouldn't have this change... what do you think?

BTW, two days ago? I didn't see the PR, I will review my mail notifications and filtering, but please, ping me personally next time (if you want review :wink: ).